pkg/server/server.go

```diff
@@ -264,13 +263,20 @@ func ensureMTLS(next http.Handler) http.Handler {
 func logRequest(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		cert := getClientCert(r)
```

Comment:
Why are we getting the client cert if it is not actually being used? We don't insist on mTLS anymore, so likely as not there won't be a client cert, or it will be useless. Either we use the mTLS client cert entirely, or not at all.

Reply:
Well, I put RemoteAddr for both cases into log now.